src/cloudwatch/modules/client/querystringbuilder.py

```diff
@@ -67,3 +69,6 @@ def _add_values(self, metric, metric_map, metric_prefix):
         metric_map[metric_prefix + self._STAT_MIN] = metric.statistics.min
         metric_map[metric_prefix + self._STAT_SUM] = metric.statistics.sum
         metric_map[metric_prefix + self._STAT_SAMPLE] = metric.statistics.sample_count
+
+    def _add_unit(self, metric, metric_map, metric_prefix):
+        metric_map[metric_prefix + self._UNIT_KEY] = metric.unit
```

Comment:
Is it possible that collectd give different unit every time? for example: sometimes, it is seconds, some times, it is miscroseconds.